commons/c2cgeoportal_commons/models/main.py

```diff
@@ -183,6 +183,72 @@ def __unicode__(self):
     schema=_schema
 )
 
+# association table role <> restriciton area
+class RAForRole(Base):
+    __tablename__ = 'role_restrictionarea'
+    __table_args__ = (
+        {"schema": _schema}
+    )
+
+    id = Column("id", Integer, primary_key=True)
+    role_id = Column("role_id", Integer, ForeignKey(_schema + ".role.id"))
+    restrictionarea_id = Column("restrictionarea_id", Integer, ForeignKey(_schema + ".restrictionarea.id"))
+
+# association table layer <> restriciton area
+layer_ra = Table(
+    "layer_restrictionarea", Base.metadata,
+    Column(
+        "layer_id", Integer,
+        ForeignKey(_schema + ".layer.id"), primary_key=True
+    ),
+    Column(
+        "restrictionarea_id", Integer,
+        ForeignKey(_schema + ".restrictionarea.id"), primary_key=True
+    ),
+    schema=_schema
+)
+
+class RestrictionArea(Base):
```

Comment:
Why moving this, is there any change?

Reply:
backref to role from restriction area was removed,
ref to RestricationArea added to Role:
https://github.com/camptocamp/c2cgeoportal/blob/a124ef3ff9752ab166f1bfc93313a15d7f519aa3/commons/c2cgeoportal_commons/models/main.py#L284